src/lib/relay/routes.tsx

```diff
@@ -54,15 +54,20 @@ class Gene extends Relay.Route {
 
 class WorksForYou extends Relay.Route {
   static queries = {
-    me: (component, params) => Relay.QL`
+    viewer: (component, params) => Relay.QL`
       query {
-        me {
-          ${component.getFragment("me", params)}
+        viewer {
+          ${(component.getFragment("viewer", params))}
```

Comment:
I think this would remove the need to use the `viewer` namespace at all in the rest of the code:

```graphql
query {
  viewer {
    me {
      ${component.getFragment("me", params)}
    }
  }
}
```